docs/build/qubit-order.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Bit-ordering in Qiskit
```

Comment:
We should use the same title in all three places (page title, metadata, and toc file). At the moment we have to take into consideration the unfortunate truncation that happens in the left nav (there is an issue out to fix that), so shorter might be better, at least temporarily.